<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>666</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>928</width>
    <height>666</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>928</width>
    <height>666</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>161</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QPushButton" name="pushButton_31">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
      </property>
      <property name="text">
       <string>基本测试</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_32">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
      </property>
      <property name="text">
       <string>加密</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_33">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
      </property>
      <property name="text">
       <string>解密</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_34">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
      </property>
      <property name="text">
       <string>暴力破解</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_35">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
      </property>
      <property name="text">
       <string>验证封闭性</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>631</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="page_16">
    <widget class="QPushButton" name="pushButton_36">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>310</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
     </property>
     <property name="text">
      <string>开始加密</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_16">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>260</y>
       <width>261</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>158</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>请输入密钥（10bit）：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_34">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>261</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_35">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>261</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>149</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>请输入明文（8bit）：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_17">
    <widget class="QLineEdit" name="lineEdit_36">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>请输入密钥（逗号分隔的整数列表）：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>请输入明文（字符串）：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_37">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_37">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>310</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
     </property>
     <property name="text">
      <string>开始加密</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_17">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>220</y>
       <width>261</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_18">
    <widget class="QTextEdit" name="textEdit_18">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>220</y>
       <width>261</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_39">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>请输入密钥（逗号分隔的整数列表）：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_38">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>310</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
     </property>
     <property name="text">
      <string>开始解密</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_38">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>请输入密文（字符串）：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_39">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_19">
    <widget class="QLineEdit" name="lineEdit_40">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_19">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>240</y>
       <width>261</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_41">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>110</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_41">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>请输入明文（字符串）：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_39">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>310</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
     </property>
     <property name="text">
      <string>开始验证</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_42">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>请输入不同的密钥1（逗号分隔的整数列表）：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_43">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>请输入不同的密钥2（逗号分隔的整数列表）：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_42">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>180</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_20">
    <widget class="QTextEdit" name="textEdit_20">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>210</y>
       <width>261</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_44">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>请输入您获取的明文（字符串）：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_40">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>300</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
     </property>
     <property name="text">
      <string>开始破解</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_43">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_45">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>请输入您获取的密文：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_44">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>120</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>S-DES加密系统</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
